app/models/post.rb

```diff
@@ -0,0 +1,4 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments # class_name: "comment ", foreign_key: "reference_id"
+end
```

Comment:
### keep your code clean! no comments unless they are for explanation, use git control if you need to check out your previews code